eng/scripts/run_tests.ps1

```diff
@@ -10,6 +10,8 @@ Param(
 $ErrorActionPreference = 'Stop'
 Push-Location sdk/$serviceDirectory
 
+$Env:GOEXPERIMENT="nocoverageredesign"
```

Comment:
@benbp let me know if there's someplace I can centralize this (see duplicate change above).

Reply:
@jhendrixMSFT you can put it in here if that works: https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/templates/variables/globals.yml

Reply:
Those are also env vars?

Reply:
@jhendrixMSFT yep anything set as a pipeline variable gets passed down to all processes as environment variables (albeit uppercased and with non-alphanumeric characters removed).